### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by gids-dadi**
- The PR has the todo.js file to add and remove list items.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1,2 @@
...
-    li.appendChild(checkBox);
-    div.appendChild(p);
-    div.appendChild(menuIcon);
-    li.appendChild(div);
-    listTodo.appendChild(line);
-    listTodo.appendChild(li);
-  });
-}
-populate();
+import './todo.js';
```
Comment (vic778) [inline src/index.js:2]: Kindly the application is not showing in the browser. try to fix it . this is the message error am getting `The resource from “http://localhost:8080/index.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).`
Screenshot from 2022-04-30 16-23-45

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @gids-dadi 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (gids-dadi) [reply src/index.js:2]: Kindly read the README file, the program runs with _npm start_

**Review by smunozmo — APPROVED** [review]
Hi @gids-dadi, 👋 

## Highlights 👏 

- The to-do list is working, nice!
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**